example/test.js

```diff
@@ -0,0 +1,9992 @@
+// source: test.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {missingRequire} reports error on implicit type usages.
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
+
+import * as jspb from 'google-protobuf';
+var goog = jspb;
```

Comment:
This probably wiill need to be something like `import { goog } from 'closure-library/goog.js`.
But that may be fine to figure out later.

Reply:
the correct import for goog is `import * as goog from 'closure-library/closure/goog';`
We still need to figure out how the jspb stuff should be imported.